actors/builtin/miner/miner_actor.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"encoding/binary"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
FYI we'll be switching to xerrors, so just use that now for new things (or at least use the `%w` format specifier for wrapping). The current state is a mix of both. #176